dockstore-integration-testing/src/test/java/io/dockstore/webservice/UserResourceIT.java

```diff
@@ -278,6 +285,15 @@ public void testSelfDestruct() throws ApiException {
         }
         assertTrue(expectedAdminAccessToFail);
 
+        // Verify that self-destruct also deleted the service
+        boolean expectedAdminServiceAccessToFail = false;
+        try {
+            adminWorkflowsApi.getWorkflowByPath(github + "/" + serviceRepo, null, true);
+        } catch (ApiException e) {
+            expectedAdminServiceAccessToFail = true;
```

Comment:
Should assert that the exception is due to it being missing (would have expected a relevant error message or a wrapped 404 or something)
Otherwise, this could "pass" if the api is just broken in some other way.